### ISSUE TITLE
Great job xNasuni!

### ISSUE THREAD

**Issue by crazyseba** [xNasuni/artlist-downloader#6]
thanks for sharing! It works great on music.
Any chance of creating something for footage on artlist? it wood be very cool.

**Comment by xNasuni** [comment]
i believe that at this point it would go too far. the purpose for me creating this isn't for piracy and me personally i only use it to listen to music outside of artlist as audio files, but i feel like footage is meant to be used as stock media in videos on youtube as an example, and not paying for a license goes against my morals.

a license on artlist doesn't cost that much and it ranges from $9.99/mo for music & sfx, and $40.00/mo for everything which for the quality that artlist provides, i feel is adequate and fair.

tldr; sorry if i didn't meet your expectations but i feel like this is actual piracy and you should consider buying a license if you want to use it in commerical videos or alike.

**Comment by crazyseba** [comment]
You don't have to meet my expectations, still great work. I don't do commercial work, Im trying to learn video editing and thats why I need footage, but no personal intension of uploading anything or sharing, only for personal practice. I do agree that otherwise you should buy a license. Thanks for this code because its also helpfull for practice of sound design
